homeassistant/components/geofency/__init__.py

```diff
@@ -136,12 +132,20 @@ def _set_location(hass, data, location_name):
     """Configure based on config entry."""
     hass.components.webhook.async_register(
         DOMAIN, 'Geofency', entry.data[CONF_WEBHOOK_ID], handle_webhook)
+
+    hass.async_create_task(
+        hass.config_entries.async_forward_entry_setup(entry, DEVICE_TRACKER)
+    )
     return True
 
 
 async def async_unload_entry(hass, entry):
     """Unload a config entry."""
     hass.components.webhook.async_unregister(entry.data[CONF_WEBHOOK_ID])
+
+    hass.async_create_task(
```

Comment:
We can await the coroutine directly. We don't need to create a task.

Reply:
Isn't better practice to create a task so that we don't block this thread? Or does it not matter since all we're doing is registering with the dispatcher?

Reply:
There is no risk of deadlock when using this method.